### DISCUSSION TITLE
Combining different outcomes in one search

### DISCUSSION THREAD

**Discussion by Margriet87** [asreview/asreview#1301] [Q&A]
This will be the third time working with ASReview, and excited to start screening again! 
For our literature review, we look at the effects of treatment for people with mild intellectual disabilities on several outcomes, including working alliance, treatment motivation and reduction of problem behavior. 
We are currently buidling the search strategy combining these terms (working alliance OR treatment motivation OR problem behavior). In ASReview we want to mark records as relevant if they address at least one of these treatment outcomes. My question is: will it confuse ASReview if we do it like this? Or does it recognize that the difference between 'working alliance' and 'treatment motivation' is conceptually much larger than the difference between 'working alliance' and 'therapeutic relationship' (different words for more or less the same concept as 'working alliance'). 
I hope someone knows, would definitely help is if we can indeed just screen all the records one time selecting those with the different relevant outcomes at once. 

Thanks!

**Comment by rohitgarud** [comment]
I am no expert in Systematic Review or the detailed workings of Asreview.. but my guess is that your selection of feature extraction algorithm will affect the results in your case.. the algorithms other than the default TF-IDF should be able to understand the difference to some extent and adjust for that.. mostly due to the similar meaning of the words alliance and relationship.. but I think experts will be able to give a better explanation.. @Margriet87
